src/widget/widget.cpp

```diff
@@ -2150,6 +2151,7 @@ void Widget::friendRequestsUpdate()
         {
             onAddClicked();
             addFriendForm->setMode(AddFriendForm::Mode::FriendRequest);
+            addFriendForm->onCurrentChanged(AddFriendForm::Mode::FriendRequest);
```

Comment:
This shouldn't be necessary, because `addFriendForm->setMode(AddFriendForm::Mode::FriendRequest);` calls ` tabWidget->setCurrentIndex(mode);` which then emits a `currentChanged` signal, which then calls `onCurrentChanged()`.